executor/src/witgen/mod.rs

```diff
@@ -398,9 +422,26 @@ impl<'a, T: FieldElement> FixedData<'a, T> {
         }
     }
 
+    pub fn filter_identities(self, f: impl Fn(&FixedData<T>, &Identity<T>) -> bool) -> Self {
```

Comment:
Please document.

Reply:
I actually would prefer only having the `with_global_range_constraints` function.

Reply:
Done.

> I actually would prefer only having the `with_global_range_constraints` function.

Me too, but this is also used to filter identities that reference later-stage stuff. And the callback needs a reference to `fixed_data`...